src/components/Badge.astro

```diff
@@ -25,10 +25,10 @@ const inverseColoursMap = {
 }
 
 const map = inverse ? inverseColoursMap : coloursMap;
-const badgeClass =`${map[type]} type-caption-2 px-2 py-0.5 rounded-full border`;
+const badgeClass =`${map[type]} align-text-top type-caption-2 px-2 rounded-full border`;
```

Comment:
@alexBinns what was the reasoning for changing this?
* I can't see a difference from having the `align-text-top` class.
* We now have less padding on the y axis (`py-0.5`)